content/20.pharmaceuticals.md

```diff
@@ -264,8 +265,9 @@ While there is clear computational and _in vitro_ support for ebselen's potentia
 In summary, N3 is a computationally designed molecule that inhibits the viral transcription through inhibiting Mpro.
 Ebselen is both a strong Mpro inhibitor and strong inhibitor of viral replication _in vitro_ that was found to reduce SARS-CoV-2 viral load even more effectively than N3.
 Ebselen is a very promising compound since its safety has been demonstrated in other indications.
-However, Ebselen is likely a false positive since it is a promiscuous compound that can have many targets [@doi:10.1039/c8cc04258f].
+However, Ebselen could prove to be a false positive since it is a promiscuous compound that can have many targets [@doi:10.1039/c8cc04258f].
```

Comment:
That makes sense, happy to change it back!
```suggestion
However, Ebselen is likely to be a false positive since it is a promiscuous compound that can have many targets [@doi:10.1039/c8cc04258f].
```